code-of-conduct-languages/hi.md

```diff
@@ -0,0 +1,30 @@
+## CNCF सामुदायिक आचार संहिता v1.0
+
+### आचार संहिता का योगदानकर्ता
+
+इस परियोजना के योगदानकर्ताओं और अनुरक्षकों के रूप में, और एक खुले और स्वागत करने वाले समुदाय को बढ़ावा देने के हित में, हम उन सभी लोगों का सम्मान करने की प्रतिज्ञा करते हैं जो रिपोर्टिंग issues के माध्यम से योगदान करते हैं, फीचर अनुरोध पोस्ट करते हैं, प्रलेखन अपडेट करते हैं, pull requset या patches, और अन्य गतिविधियों में भाग लेते हैं।
```

Comment:
I think you can change a bit here, take योगदान करते हैं to the last of the sentence as it is addressing all the examples and not only the reporting issues one.
Like this -> प्रतिज्ञा करते हैं जो issues रिपोर्टिंग के माध्यम से , फीचर अनुरोध पोस्ट करने के माध्यम से , प्रलेखन अपडेट करने के माध्यम से, pull request या patches, और अन्य गतिविधियों में भाग लेने के माध्यम से योगदान करते हैं।